README.md

```diff
@@ -84,13 +84,21 @@ has to adjust the systemd service file:
           -i \
           /usr/lib/dracut/modules.d/46sshd/sshd.service
 
-Make sure that `/root/.ssh/authorized_keys` contains the right
-keys, as it's included in the [initramfs][iramfs]:
+`/root/.ssh/dracut_authorized_keys` is the default `authorized_keys` file.
+
+A custom located authorized_keys file can be deployed as
+`/etc/dracut-sshd/authorized_keys`. This might be needed for example
+when using an [rpm-ostree]-based distro and your root directory setup
+differs from the long-time standard.
+Make sure that your `authorized_keys` file contains the right keys, as
+it's included in the [initramfs][iramfs]:
```

Comment:
This starts a bit abruptly. What about something like this:

```
Dracut-sshd includes the first available ssh authorized keys files of the following list into the initramfs:

- ...
- ...
- ...

Note that on Fedora Silverblue the `/etc/dracut-sshd` location has to be used due to ...

Thus make sure that either `/root/.ssh/authorized_keys` contains the right keys or create one of the alternative keys files.

```

Reply:
Corrected